lib/unlock-helpers.js

```diff
@@ -6,13 +6,21 @@ const PIN_UNLOCK = "pin";
 const PASSWORD_UNLOCK = "password";
 const PATTERN_UNLOCK = "pattern";
 const FINGERPRINT_UNLOCK = "fingerprint";
-const UNLOCK_TYPES = [PIN_UNLOCK, PASSWORD_UNLOCK, PATTERN_UNLOCK, FINGERPRINT_UNLOCK];
+const SWIPE_UNLOCK = "swipe";
+const UNLOCK_TYPES = [PIN_UNLOCK, PASSWORD_UNLOCK, PATTERN_UNLOCK, FINGERPRINT_UNLOCK, SWIPE_UNLOCK];
 const KEYCODE_NUMPAD_ENTER = "66";
 const UNLOCK_WAIT_TIME = 100;
 const HIDE_KEYBOARD_WAIT_TIME = 100;
 const INPUT_KEYS_WAIT_TIME = 100;
 
 let helpers = {};
+
+helpers.PIN_UNLOCK = PIN_UNLOCK;
+helpers.PASSWORD_UNLOCK = PASSWORD_UNLOCK;
+helpers.PATTERN_UNLOCK = PATTERN_UNLOCK;
+helpers.FINGERPRINT_UNLOCK = FINGERPRINT_UNLOCK;
+helpers.SWIPE_UNLOCK = SWIPE_UNLOCK;
```

Comment:
Searching around in the code, I don't see these used anywhere else. Am I missing something? If not, this can get removed. If it's needed, it can be simplified to 
```js
let helpers = {
  PIN_UNLOCK,
  PASSWORD_UNLOCK,
  PATTERN_UNLOCK,
  FINGERPRINT_UNLOCK,
  SWIPE_UNLOCK,
};
```